test-init.gradle

```diff
@@ -0,0 +1,45 @@
+initscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath files(
+            '/root/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bc-fips/2.0.0/ee9ac432cf08f9a9ebee35d7cf8a45f94959a7ab/bc-fips-2.0.0.jar',
+            '/root/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcpkix-fips/2.0.7/1eea0f325315ca6295b0a6926ff862d8001cdf9/bcpkix-fips-2.0.7.jar',
+            '/root/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bctls-fips/2.0.19/9cc33650ede63bc1a8281ed5c8e1da314d50bc76/bctls-fips-2.0.19.jar',
+            '/root/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcutil-fips/2.0.3/a1857cd639295b10cc90e6d31ecbc523cdafcc19/bcutil-fips-2.0.3.jar'
+        )
+    }
+}
+
+gradle.taskGraph.whenReady { graph ->
```

Comment:
should we wrap these tasks instead of changing them, or only execute this bit when an environment variable is set (and set that var in our `Dockerfile`)?

Reply:
I think so. I put up an attempt to do that but its not quite there yet. I do think your suggestion will be the way to go though in the end.